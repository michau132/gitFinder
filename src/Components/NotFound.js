import React from 'react';
import { Typography } from '@material-ui/core';

const NotFound = () => (
  <Typography variant="h5">
      User that you are looking for not exist
  </Typography>
);

export default NotFound;
