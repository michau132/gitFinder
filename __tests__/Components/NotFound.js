import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NotFound from '../../src/Components/NotFound';

describe('testing NotFound Component', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<NotFound />);
  });

  test('renders without crashing', () => {
    expect(wrapper).toBeDefined();
  });

  test('should match snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
