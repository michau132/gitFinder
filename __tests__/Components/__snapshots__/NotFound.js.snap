// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing NotFound Component should match snapshot 1`] = `
<WithStyles(Typography)
  variant="h5"
>
  User that you are looking for not exist
</WithStyles(Typography)>
`;
